website/docs/concepts/contexts.mdx

```diff
@@ -1,12 +1,25 @@
 ---
 title: "Contexts"
 sidebar_label: Contexts
-description: "Using contexts to pass data within application"
+description: "Using contexts to pass deeply nested data"
 ---
 
-Generally data is passed down the component tree using props but that becomes tedious for values such as
-user preferences, authentication information etc. Consider the following example which passes down the
-theme using props:
+Usually, data is passed from a parent component to a child component via props.
+But passing props can become verbose and annoying if you have to pass them through many components in the middle,
```

Comment:
"in the middle" middle of what?
Would be nice to have a visual of some sort of components tree here :/

Reply:
Not blocking though

Reply:
In middle of component that owns the data and one that needs to consume it. How else would you word it?

Reply:
I'm going to merge this in this. Please an create issue for the visual so we can take care of it separately